packages/openid4vc/package.json

```diff
@@ -10,26 +10,31 @@
   "publishConfig": {
     "access": "public"
   },
-  "homepage": "https://github.com/hyperledger/aries-framework-javascript/tree/main/packages/openid4vc-client",
+  "homepage": "https://github.com/hyperledger/aries-framework-javascript/tree/main/packages/openid4vc",
   "repository": {
     "type": "git",
     "url": "https://github.com/hyperledger/aries-framework-javascript",
-    "directory": "packages/openid4vc-client"
+    "directory": "packages/openid4vc"
   },
   "scripts": {
     "build": "yarn run clean && yarn run compile",
     "clean": "rimraf ./build",
     "compile": "tsc -p tsconfig.build.json",
     "prepublishOnly": "yarn run build",
-    "test": "jest"
+    "test": "jest --forceExit --detectOpenHandles"
```

Comment:
Was this just required for debugging or do we always need it for this?

Reply:
No not needed